salt/modules/ddns.py

```diff
@@ -191,16 +202,25 @@ def update(zone, name, ttl, rdtype, data, nameserver='127.0.0.1', timeout=5,
                     is_exist = True
                     break
 
+    log.debug('record exists: %s', is_exist)
```

Comment:
A bit of a nit-pick but can you make the `r` in `record` upper case please?

Reply:
Yup! My bad :)